### PR TITLE
Automation of the avoid ballooning client_mount_timeout(BZ#2233800)

### PR DESCRIPTION
**Pull request by SrinivasaBharath**
# Description
Automation of the following scenario:- 
1. From the client node blocked the outgoing connection to the MONs. 
   Command used: iptables -A OUTPUT -d <Mon-IP>  -j DROP
   Example on Client node:
     [root@ceph-7-0-zmjrew-node7 ceph]# iptables -A OUTPUT -d 10.0.210.81  -j DROP

3. Executed any rados cli command.
    Example: rados df
4. Calculate the timeout of the command. The timeout should be 5 minutes.

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2233800
Polarion test case link:   https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83584004

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by SrinivasaBharath** [comment]
Test execution pass logs-
             1. Reef: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-I46CSR
             2. Quincy : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-4S5J9N

**Comment by SrinivasaBharath** [comment]
Execution test logs:
 1. Reef log: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-SYCALF
 2. Quincy log : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-25Z8E3

File: tests/rados/test_rados_ballooning_clinet.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+"""
```
Comment (neha-gangadhar) [inline tests/rados/test_rados_ballooning_clinet.py:1]: please fix minor typo in file name - "client"

File: suites/quincy/rados/tier-2_rados_test-bugfixes.yaml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+          - ceph-common
+        copy_admin_keyring: true          # Copy admin keyring to node
+        caps:                             # authorize client capabilities
+          mon: "allow *"
+          osd: "allow *"
+          mds: "allow *"
+          mgr: "allow *"
+  - test:
+      name: Verification of the Mon ballooning
+      module: test_rados_ballooning_clinet.py
```
Comment (neha-gangadhar) [inline suites/quincy/rados/tier-2_rados_test-bugfixes.yaml:110]: please fix minor typo - "client"

File: suites/reef/rados/tier-2_rados_test-bugfixes.yaml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+          - ceph-common
+        copy_admin_keyring: true          # Copy admin keyring to node
+        caps:                             # authorize client capabilities
+          mon: "allow *"
+          osd: "allow *"
+          mds: "allow *"
+          mgr: "allow *"
+  - test:
+      name: Verification of the Mon ballooning
+      module: test_rados_ballooning_clinet.py
```
Comment (neha-gangadhar) [inline suites/reef/rados/tier-2_rados_test-bugfixes.yaml:110]: please fix minor typo - "client"

File: tests/rados/test_rados_ballooning_clinet.py (hunk 1 above)
Comment (SrinivasaBharath) [reply tests/rados/test_rados_ballooning_clinet.py]: Typo fixed in the file name.

File: suites/quincy/rados/tier-2_rados_test-bugfixes.yaml (hunk 2 above)
Comment (SrinivasaBharath) [reply suites/quincy/rados/tier-2_rados_test-bugfixes.yaml]: Fixed the issue.

File: suites/reef/rados/tier-2_rados_test-bugfixes.yaml (hunk 3 above)
Comment (SrinivasaBharath) [reply suites/reef/rados/tier-2_rados_test-bugfixes.yaml]: Fixed the issue.

File: suites/quincy/rados/tier-2_rados_test-bugfixes.yaml [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+        node: node7                       # client node
+        install_packages:
+          - ceph-common
+        copy_admin_keyring: true          # Copy admin keyring to node
+        caps:                             # authorize client capabilities
+          mon: "allow *"
+          osd: "allow *"
+          mds: "allow *"
+          mgr: "allow *"
+  - test:
```
Comment (pdhiran) [inline suites/quincy/rados/tier-2_rados_test-bugfixes.yaml:108]: please add the test to enable logging to file on cluster. will be helpful if we need to debug on cluster.

File: suites/reef/rados/tier-2_rados_test-bugfixes.yaml [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+        id: client.1                      # client Id (<type>.<Id>)
+        node: node7                       # client node
+        install_packages:
+          - ceph-common
+        copy_admin_keyring: true          # Copy admin keyring to node
+        caps:                             # authorize client capabilities
+          mon: "allow *"
+          osd: "allow *"
+          mds: "allow *"
+          mgr: "allow *"
```
Comment (pdhiran) [inline suites/reef/rados/tier-2_rados_test-bugfixes.yaml:107]: Same as above.

**Review by pdhiran — COMMENTED** [review]
Looks good with the suite changes.

**Review by pdhiran — APPROVED** [review]
looks good.

File: suites/quincy/rados/tier-2_rados_test-bugfixes.yaml (hunk 4 above)
Comment (SrinivasaBharath) [reply suites/quincy/rados/tier-2_rados_test-bugfixes.yaml:116]: Added the enable logging to file

File: suites/reef/rados/tier-2_rados_test-bugfixes.yaml (hunk 5 above)
Comment (SrinivasaBharath) [reply suites/reef/rados/tier-2_rados_test-bugfixes.yaml:107]: Added the enable logging to file.

**Review by neha-gangadhar — APPROVED** [review]
approved

**Pull request merged**